### DISCUSSION TITLE
speclib.mbr.hdf vs speclib.hdf

### DISCUSSION THREAD

**Discussion by LoayJabre** [MannLabs/alphadia#408] [Questions / Support] [answered]
I see two spectral library files generated after I do a first pass search using a FASTA file - speclib.mbr.hdf and speclib.hdf. I'm not really sure what the difference is between them, but I've seen in the comments that a second pass search can be done using the mbr.hdf file. 

Can someone explain the difference between the two spectral library files and when one should be used instead of the other?

**Comment by GeorgWa** [comment, marked as answer]
Hi, yes sure:

`speclib.hdf` is the input library which was used for the search. This can be the fully predicted library generated from fasta or a library which was loaded from hdf or tsv and maybe reannotated, re-predicted etc.

`speclib.mbr.hdf` is the output library containing all precursors identified in the sample. If a two-step search with MBR is done, this will be the inout library to the second search step.

`speclib.transfer.hdf` is the training data for the transfer learning. The precursors should be similar to `speclib.mbr.hdf` but it will contain all requested fragment types and observed mzs and intensities rather than predicted ones.
